lib/hls/hls_parser.js

```diff
@@ -4947,11 +4951,15 @@ shaka.hls.HlsParser = class {
     const uri = drmTag.getRequiredAttrValue('URI');
     const parsedData = shaka.net.DataUriPlugin.parseRaw(uri.split('?')[0]);
 
-    // The data encoded in the URI is a PSSH box to be used as init data.
-    const pssh = shaka.util.BufferUtils.toUint8(parsedData.data);
+    const psshKey = 'widevine_' + parsedData.data;
```

Comment:
There is no need to store widevine and wiseplay separately. PSSH string will be converted to the same typed array anyway.